From: David Steele <dsteele@gmail.com>
Date: Fri, 25 Sep 2015 22:37:06 -0400
Subject: Allow-older-python-requests

Remove the version requirement on python-requests in setup.py. The tight
requirement caused deb-packaged versions to fail on older distributions.

===================================================================
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 3403caf..5c754e4 100644
--- a/setup.py
+++ b/setup.py
@@ -29,7 +29,7 @@ setup(
         ],
     },
     install_requires=[
-        'requests >= 2.7.0',
+        'requests',
         'pycups',
     ],
     extras_require={
